casanovo/denovo/model_runner.py

```
@@ -350,7 +350,9 @@ def predict(
         if evaluate:
             self.log_metrics(predict_dataloader)
 
-    def initialize_trainer(self, train: bool) -> None:
+    def initialize_trainer(
+        self, train: bool, enable_progress_bar: bool = True
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)